docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -1,66 +1,49 @@
-* Ensure that the database migration has completed. For more information, see xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances].
-* Ensure that {OpenShift} nodes where `manila-share` service will be deployed
-can reach the management network that the storage system is in.
-* If the deployment uses CephFS through NFS as a storage backend, ensure that
-a new clustered Ceph NFS service is deployed on the Ceph cluster with the help
-of Ceph orchestrator. For more information, see
-xref:creating-a-ceph-nfs-cluster_migrating-databases[Creating a Ceph NFS cluster].
-* Ensure that services such as {identity_service_first_ref} and memcached are available prior to
-adopting the Shared File Systems services.
-* If tenant-driven networking was enabled (`driver_handles_share_servers=True`),
-ensure that {networking_first_ref} has been deployed prior to adopting Shared File Systems services.
+* The {rhos_component_storage_file} systemd services such as `api`, `cron`, and `scheduler` are stopped. For more information, see xref:stopping-openstack-services_migrating-databases[Stopping {rhos_prev_long} services].
+* If the deployment uses CephFS through NFS as a storage back end, the Pacemaker ordering and collocation constraints are adjusted. For more information, see xref:stopping-openstack-services_migrating-databases[Stopping {rhos_prev_long} services].
+* The {rhos_component_storage_file} Pacemaker service (`openstack-manila-share`) is stopped. For more information, see xref:stopping-openstack-services_migrating-databases[Stopping {rhos_prev_long} services].
+* The database migration is complete. For more information, see xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances].
+* The {OpenShift} ({OpenShiftShort}) nodes where the `manila-share` service is to be deployed can reach the management network that the storage system is in.
+* If the deployment uses CephFS through NFS as a storage back end, a new clustered Ceph NFS service is deployed on the {CephCluster} cluster with the help
+of Ceph orchestrator. For more information, see xref:creating-a-ceph-nfs-cluster_migrating-databases[Creating a Ceph NFS cluster].
+* Services such as the {identity_service_first_ref} and memcached are available prior to adopting the Shared File Systems services.
+* If you enabled tenant-driven networking by setting `driver_handles_share_servers=True`, the {networking_first_ref} is deployed.
 
 .Procedure
 ifeval::["{build}" != "downstream"]
 . Define the `CONTROLLER1_SSH` environment variable, if it link:stop_openstack_services.md#variables[hasn't been
 defined] already. Then copy the configuration file from {OpenStackShort} {rhos_prev_ver} for reference.
 endif::[]
 ifeval::["{build}" != "upstream"]
-. Define the `CONTROLLER1_SSH` environment variable, if it hasn't been
+. Define the `CONTROLLER1_SSH` environment variable, if it has not been
```

Comment:
How do customers know whether `CONTROLLER1_SSH` has already been defined? How would they define it if it isn't?

Reply:
@gouthampacha ^

Reply:
@gouthampacha Can you please review the comments above and provide your input?

Reply:
Since this PR has been peer reviewed, I'd like to merge asap. We could apply whatever change we think is necessary in another PR.

Reply:
Sorry about missing this :( I don't do well with GitHub reviews.

As Francesco called out, this variable "CONTROLLER1_SSH" is declared in earlier sections ("Stopping OpenStack services", "Migrating databases to the control plane", ..) 
We've linked the place to go to right above too; so this needs a minor rewrite:

```
ifeval::["{build}" != "downstream"]
. Define the `CONTROLLER1_SSH` environment variable, if it link:stop_openstack_services.md#variables[hasn't been
defined] already. Then copy the configuration file from {OpenStackShort} {rhos_prev_ver} for reference.
endif::[]
ifeval::["{build}" != "upstream"]
. Define the `CONTROLLER1_SSH` environment variable, to point to the RHOSP Controller node. For example:

$ CONTROLLER1_SSH="ssh -i <path to SSH key> root@<node IP>"

Then copy the configuration file from {OpenStackShort} {rhos_prev_ver} for reference.
endif::[]
```